server/routes/contacts/create/enter-dob/createContactEnterDobController.ts

```diff
@@ -2,11 +2,8 @@ import { Request, Response } from 'express'
 import { Page } from '../../../../services/auditService'
 import { PageHandler } from '../../../../interfaces/pageHandler'
 import { CreateContactEnterDobSchemaType } from './createContactEnterDobSchemas'
-import { ContactsService } from '../../../../services'
 
```

Comment:
I see what you mean about moving the create point further down the journey - hence the early implementation of check your answers. Lightbulb.